lib/CL/devices/tbb/tbb.h

```diff
@@ -0,0 +1,46 @@
+/* OpenCL device using the Intel TBB library (derived from the pthread device).
+
+   Copyright (c) 2011 Universidad Rey Juan Carlos
```

Comment:
Check the copyright.

Reply:
Addressed in 2933f1eac17bad78390f97cbdd47e413191706c2.